global_install_scripts/update_talisman.bash

```diff
@@ -7,134 +7,159 @@ FORCE_DOWNLOAD=${FORCE_DOWNLOAD:-''}
 
 function run() {
 
-    # Download appropriate (appropriate = based on OS and ARCH) talisman binary from github
-    # Copy the talisman binary to $TALISMAN_SETUP_DIR ($HOME/.talisman/bin)
-
-    declare TALISMAN_BINARY_NAME
-
-    IFS=$'\n'
-    VERSION=${VERSION:-'latest'}
-    INSTALL_ORG_REPO=${INSTALL_ORG_REPO:-'thoughtworks/talisman'}
-
-    TALISMAN_SETUP_DIR=${HOME}/.talisman/bin           # location of central install: talisman binary and hook script
-
-    TEMP_DIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'talisman_setup')
-    #trap "rm -r ${TEMP_DIR}" EXIT
-    chmod 0700 ${TEMP_DIR}
-
-    function echo_error() {
-	echo -ne $(tput setaf 1) >&2
-	echo "$1" >&2
-	echo -ne $(tput sgr0) >&2
-    }
-    export -f echo_error
-
-    function echo_debug() {
-	[[ -z "${DEBUG}" ]] && return
-	echo -ne $(tput setaf 3) >&2
-	echo "$1" >&2
-	echo -ne $(tput sgr0) >&2
-    }
-    export -f echo_debug
-
-    function echo_success {
-	echo -ne $(tput setaf 2)
-	echo "$1" >&2
-	echo -ne $(tput sgr0)
-    }
-    export -f echo_success
-
-    function collect_version_artifact_download_urls() {
-	curl -Ls -w %{url_effective} "https://github.com/${INSTALL_ORG_REPO}/releases/latest" | grep -Eo '/'${INSTALL_ORG_REPO}'/releases/download/.+/[^/"]+' | sed 's/^/https:\/\/github.com/' > ${TEMP_DIR}/download_urls
-	echo_debug "All release artifact download urls can be found at ${TEMP_DIR}/download_urls:"
-	[[ -z "${DEBUG}" ]] && return
-	cat ${TEMP_DIR}/download_urls
+	# Download appropriate (appropriate = based on OS and ARCH) talisman binary from github
+	# Copy the talisman binary to $TALISMAN_SETUP_DIR ($HOME/.talisman/bin)
+
+	declare TALISMAN_BINARY_NAME
+
+	IFS=$'\n'
+	VERSION=${VERSION:-'latest'}
+	INSTALL_ORG_REPO=${INSTALL_ORG_REPO:-'thoughtworks/talisman'}
+
+	TALISMAN_SETUP_DIR=${HOME}/.talisman/bin # location of central install: talisman binary and hook script
+
+	TEMP_DIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'talisman_setup')
+	#trap "rm -r ${TEMP_DIR}" EXIT
+	chmod 0700 ${TEMP_DIR}
+
+	function echo_error() {
+		echo -ne $(tput setaf 1) >&2
+		echo "$1" >&2
+		echo -ne $(tput sgr0) >&2
+	}
+	export -f echo_error
```

Comment:
Is this just a formatting change ?
It is very difficult to understand the scope of changes when a commit includes both formatting change and code changes.
I suggest splitting PR's into re-formatting ones and actual code change ones.

Reply:
It is formatting issue. The indentations and formatting has been corrected automatically on save by VS code. The only changes that installation, update and uninstallation scripts actually have is the talisman environment variables setting for proper installation and uninstallation. I Will take care of this next time. 
Respective files changed along with line numbers are mentioned below.

install.bash: 226-298
update_talisman.bash: 136-161
uninstall.bash: 104-131